src/app/components/popup-host/popup-host.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="popup" [ngClass]="{visible: isPopup()}">
```

Comment:
Я всегда рекомендую подход с биндингом классов, он читается как то попроще и биндинг более нативная история, Ангуляру чуть проще ее оптимизировать и мигрировать на новые инструменты
```suggestion
<div class="popup" [class. visible]="isPopup()">
```